packages/server/modules/comments/services/index.ts

```diff
@@ -176,15 +169,23 @@ export async function createCommentReply({
 
   const [newComment] = await Comments().insert(comment, '*')
   try {
-    const commentLink: Omit<CommentLinkRecord, 'commentId'> = {
+    const commentLink: CommentLinkRecord = {
       resourceId: parentCommentId,
-      resourceType: 'comment'
+      resourceType: 'comment',
+      commentId: newComment.id
     }
-    await module.exports.streamResourceCheck({
+    await streamResourceCheckFactory({
+      checkStreamResourceAccess: checkStreamResourceAccessFactory({ db })
+    })({
       streamId,
-      resources: [commentLink]
+      resources: [
```

Comment:
Main reason why I have to change stuff is because the previous implementation had broken types (stuff typed as any)